sonar-kotlin-plugin/src/test/java/org/sonarsource/kotlin/checks/TabsCheckTest.kt

```diff
@@ -19,23 +19,4 @@
  */
 package org.sonarsource.kotlin.checks
 
-import org.junit.jupiter.api.Test
-import org.sonarsource.kotlin.verifier.KotlinVerifier
-
-class TabsCheckTest {
-
-    @Test
-    fun test() {
-        KotlinVerifier(TabsCheck()) {
-            fileName = "TabsCheckSample.kt"
-        }.verify()
-    }
-
-    @Test
-    fun `no issues`() {
-        KotlinVerifier(TabsCheck()) {
-            fileName = "DummyNoIssues.kt"
-        }.verifyNoIssue()
-    }
-
-}
+class TabsCheckTest : CheckTestWithNoSemantics(TabsCheck(), sampleFileNoSemantics = "DummyNoIssues.kt")
```

Comment:
Sorry, just noticed. In this test "DummyNoIssues.kt" file is not used to test the check without semantic.
This rule doesn't care about semantic at all.

The second test is a "happy path", to test when the issue is not raised. This cannot be tested using one file, because this rule raises an issue on the whole file.

While technically using CheckTestWithNoSemantics here and having a test without semantic is not harmful, it's a little bit confusing, because the purpose is different

Reply:
Ah okay I see, no problem, I'll just add the test separately